src/web/components/AdSlot.tsx

```diff
@@ -147,12 +147,10 @@ const mobileStickyAdStyles = css`
 
 const AdSlotLabelToggled: React.FC = () => (
 	<div
-		className={cx(
-			'ad-slot__label',
-			'ad-slot__label--toggle',
-			'hidden',
-			adSlotLabelStyles,
+		className={['ad-slot__label', 'ad-slot__label--toggle', 'hidden'].join(
```

Comment:
string literals cannot exists in `css` and need to be spun out into thier own `className`